src/components/Settings/Settings.react.js

```diff
@@ -1031,7 +1031,7 @@ class Settings extends Component {
               </div>
             </div>
             <div className="img-upld">
-              <div className="label" style={{ marginBottom: '0' }}>
+              <div className="label" style={{ marginTop: '0' }}>
```

Comment:
Either use the style object, or use the css class. Avoid putting styles directly like this.